scripts/build_deb.sh

```diff
@@ -44,45 +46,45 @@ DEFAULT_RELEASE_NETWORK=$(./scripts/compute_branch_release_network.sh "${DEFAULT
 PKG_ROOT=$(mktemp -d)
 trap "rm -rf $PKG_ROOT" 0
 
-mkdir -p ${PKG_ROOT}/usr/bin
+mkdir -p "${PKG_ROOT}"/usr/bin
```

Comment:
The double quotes need to end after the bin, right ? i.e.
`mkdir -p "${PKG_ROOT}/usr/bin"`

Reply:
It doesn't need to be.